src/System.Private.ServiceModel/tests/Common/src/project.json

```diff
@@ -1,55 +1,10 @@
 {
   "dependencies": {
-    "System.Collections": "4.0.10-beta-*",
-    "System.Collections.Concurrent": "4.0.10-beta-*",
-    "System.Collections.NonGeneric": "4.0.0-beta-*",
-    "System.Collections.Specialized": "4.0.0-beta-*",
-    "System.ComponentModel": "4.0.0-beta-*",
-    "System.ComponentModel.EventBasedAsync": "4.0.10-beta-*",
-    "System.Diagnostics.Contracts": "4.0.0-beta-*",
-    "System.Diagnostics.Debug": "4.0.10-beta-*",
-    "System.Diagnostics.Tools": "4.0.0-beta-*",
-    "System.Globalization": "4.0.10-beta-*",
-    "System.IO": "4.0.10-beta-*",
-    "System.IO.Compression": "4.0.0-beta-*",
-    "System.Linq": "4.0.0-beta-*",
-    "System.Linq.Expressions": "4.0.10-beta-*",
-    "System.Linq.Queryable": "4.0.0-beta-*",
-    "System.Net.Http": "4.0.0-beta-*",
-    "System.Net.NameResolution": "4.0.0-beta-*",
-    "System.Net.Primitives": "4.0.10-beta-*",
-    "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10-beta-*",
```

Comment:
Okay - this was something I anticipated but hadn't yet fixed (I was removing everything first and then adding back dependencies as needed).